src/style.css

```
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: "Roboto";
+  src: url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+}
+
+* {
+  --color-moyo: #00acdc;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```

```suggestion
  font-family: Roboto, sans-serif;
```
Якщо шрифт з одного слова - лапки не треба